app/login/components/profile-tables.tsx

```diff
@@ -35,10 +35,22 @@ export const ProfileVisualizationsTable = (
 
   return (
     <Box className={rootClasses.sectionContent}>
-      <Typography variant="h2">My visualizations</Typography>
+      <Typography variant="h2" sx={{ mb: 4 }}>
+        My visualizations
+      </Typography>
+      <Typography variant="h3" sx={{ mb: 2 }}>
+        My published visualizations
+      </Typography>
```

Comment:
You might need translations here ?

Reply:
Yup, added them in following commit 🇩🇪 🇫🇷 🇮🇹